va/va.h

```diff
@@ -3722,6 +3722,21 @@ VAStatus vaSyncSurface (
     VASurfaceID render_target
 );
 
+/**
+ * This function blocks during specified timeout (in milliseconds) until
```

Comment:
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/include/uapi/drm/i915_drm.h?h=v5.7-rc2#n1516

It accepts timeout in nanoseconds. I think we should not lower timeout precision. I would expose timeout in nanoseconds and make argument uint64_t or int64_t. What will be infinite timeout? should we have -1 and int64_t?

Reply:
Thanks! Good point. I'll address.
What do you think about the following aspect: should we highlight that if underlying kmd/hw tasks are still in progress, vaSyncSurface2 can wait more that timeout specified (similar to sleep/usleep)?